markov.py

```diff
@@ -60,6 +61,16 @@ def remove_messages(message):
     bot.reply_to(message, 'u r not an admin 🤔')
 
 
+@bot.message_handler(commands=['version'])
+def get_repo_version(message):
+	hash_len = 7
+	try:
```

Comment:
the try block is not necessary. 
I think just the bot.reply_to(message, environ.get('HEROKU_SLUG_COMMIT')[:hash_len]) is fine.
in fact you could define the method like this:
```
hash_len = 7
commit_hash = environ.get('HEROKU_SLUG_COMMIT')[:hash_len]
bot.reply_to(message, commit_hash)
```

Reply:
the .get method should also return a empty string as default, in case there is no env var called HEROKU_SLUG_COMMIT

Reply:
like `environ.get('HEROKU_SLUG_COMMIT', '')[:hash_len]`